src/firecracker/src/api_server_adapter.rs

```diff
@@ -121,7 +121,7 @@ pub fn run_with_api(
         .try_clone()
         .expect("Failed to clone API event FD");
 
-    let api_seccomp_filter = seccomp_filter.clone();
+    let api_seccomp_filter = seccomp_filters.remove("Api").unwrap();
```

Comment:
> Does this mean the seccomp_filters will always contain the "Api" thread seccomp filtering information, either specified by users or by Firecracker, through a default seccomp filter?
Yes. This check is done on the `get_custom_filter` method in the `default_syscalls` module.
I agree that `expect` would be better for this case.

Reply:
In regards to the names of the thread categories, I prefer `Api`, `Vmm` and `Vcpu`. It would be redundant to embed the `fc_*` prefix in the naming scheme. Also, `firecracker` instead of `Vmm` may be confusing, since all threads belong to the Firecracker process.
Naming things is hard 😄